### DISCUSSION TITLE
Does RT support non-RGGB CFAs?

### DISCUSSION THREAD

**Discussion by pchambre** [Beep6581/RawTherapee#6746] [Q&A]
I just tried to load some Canon raw files (.CRW) from a PowerShot G1, and they look wrong/bad in RT. This camera uses a CYMG CFA, rather than RGGB.

Is there some way in the UI I can specify a different CFA layout?

If not, which class would I have to implement to add such support?

Thanks,
Paul

**Comment by Lawrence37** [comment]
See #5849. You'll need to start with the demosaicers.
